addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -311,6 +312,7 @@ <H2 id="id-10027">Information Disclosure: Suspicious Comments</H2>
 Analyzes web content to identify comments which contain potentially sensitive details. Which may lead to 
 further attack or exposure of unintended data.
 <p><strong>Note:</strong> The strings to look for can be extended by using the Custom Payloads add-on.
```

Comment:
While addressing the conflict the `br` was dropped, just saying in case it was accidentally (looks fine either way for me).

Reply:
Thanks, tweaked.